compiler/dyno/include/chpl/uast/Variable.h

```diff
@@ -98,6 +99,25 @@ class Variable final : public VarLikeDecl {
   bool isConfig_;
   bool isField_;
 
+  void setInitExprForConfig(AstNode* ie) {
+    if (this->initExpressionChildNum_ > -1) {
+      // have an existing initExpr, swap it
+      auto newInit = toOwned(ie);
+      this->children_[this->initExpressionChildNum_].swap(newInit);
+    } else if (this->typeExpressionChildNum_ > -1 || this->attributesChildNum() > -1) {
+      // no initExpr, but do have either typeExpr or attribute, append children
+      initExpressionChildNum_ = children_.size();
+      children_.push_back(toOwned(ie));
+      assert(numChildren() > 1);
+    } else {
+      // no initExpr and no typeExpr nor attribute
+      initExpressionChildNum_ = children_.size();
+      children_.push_back(toOwned(ie));
+      assert(numChildren() == 1);
```

Comment:
since this `else` block is the same as the preceeding `if` except for the assert, can you remove the previous block and add a conditional for what to assert?